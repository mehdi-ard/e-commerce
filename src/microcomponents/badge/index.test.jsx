import React from "react";
import renderer from "react-test-renderer";
import Badge from ".";

it("renders default", () => {
  expect(
    renderer.create(<Badge badgeContent="12">default</Badge>).toJSON()
  ).toMatchSnapshot();
});
