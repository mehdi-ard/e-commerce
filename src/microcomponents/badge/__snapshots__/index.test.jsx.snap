// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default 1`] = `
<span
  className="MuiBadge-root Badge-margin-1"
>
  default
  <span
    className="MuiBadge-badge"
  >
    12
  </span>
</span>
`;
